packages/web/src/components/typography/body.tsx

```diff
@@ -0,0 +1,47 @@
+import styled, { createGlobalStyle, css } from 'styled-components';
+
+// TODO
+// - Spacing
+// - Links with state
+
+// darkFont defines font color
+// Most of the app in on dark background and thus has bright font
+// If text should be black/dark
+
+// Articles, longer text
+const Body1 = styled.p<{ darkFont: boolean }>`
+  color: ${(props) =>
+    props.darkFont ? props.theme.colors.black.primary : props.theme.colors.white.primary};
+  fontsize: ${(props) => props.theme.fontSizes.body1.size};
```

Comment:
I like to get rid of `props` variable all together like this:
```suggestion
  font-size: ${{ theme }) => theme.fontSizes.body1.size};
```